### PR TITLE
feat: standalone version of the tuttifrutti website inserted into html/tuttifrutti

### PR DESCRIPTION
**Pull request by oricdev**
Standalone version (pure html/CSS/JS) which can be called in the URL with parameters like this: ../index.html?barcode=xxx&country=xxx&store=xxx (parameters must compliant to well-formed URL).

**Related issues and discussion:** #3871
https://github.com/openfoodfacts/openfoodfacts-server/issues/3871

### CODE REVIEW

File: html/tuttifrutti/resources/images/ico_donate_2.png [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```
Comment (hangy) [inline html/tuttifrutti/resources/images/ico_donate_2.png:1]: Not a png?

Comment (oricdev) [reply html/tuttifrutti/resources/images/ico_donate_2.png]: Item removed + section removed from index.html

**Comment by stephanegigandet** [comment]
@oricdev sorry that this never god merged and deployed. I'm closing it as a lot of things have changed.

**Comment by oricdev** [comment]
"Kein Thema!" :). Was a pleasure to participate to this great project. Wish the team the best for the future!

**Pull request closed without merging**